android/app/build.gradle

```diff
@@ -232,11 +232,31 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
     implementation "androidx.annotation:annotation:1.1.0"
-    implementation project(':@mauron85_react-native-background-geolocation-common')
+    gpsImplementation project(':@mauron85_react-native-background-geolocation-common')
```

Comment:
💯 does this mean that mauron85 only builds on gps!?

Reply:
Yessir! I'll let @schmecs confirm as she wrote it